Dockerfile

```diff
@@ -1,4 +1,6 @@
-FROM ruby:2.7-slim
+FROM ruby:2.7.3
```

Comment:
We need to keep the non-specific version of the container, as recommended by security.

Reply:
```suggestion
FROM ruby:2.7-slim
```